.changelog/v0.37.0/improvements/634-mempool-metric-times-tx-was-received.md

```diff
@@ -0,0 +1,3 @@
+- `[mempool]` Add two metrics to measure how many times a tx was received (a
```

Comment:
This shouldn't be added directly to the v0.37.0 changelog, since we've already released that version.

Which version(s) of the software are you targeting? If it's just `main`, then a changelog entry in `.changelog/unreleased/improvements` is fine. Otherwise please don't add a changelog to this PR, but add it instead on the backport branch you're targeting.